x-pack/plugins/apm/public/context/license/invalid_license_notification.tsx

```diff
@@ -9,11 +9,20 @@ import { EuiButton, EuiEmptyPrompt } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import React from 'react';
 import { useKibanaUrl } from '../../hooks/use_kibana_url';
+import { useApmPluginContext } from '../apm_plugin/use_apm_plugin_context';
 
 export function InvalidLicenseNotification() {
-  const manageLicenseURL = useKibanaUrl(
+  const {
+    plugins: { licenseManagement },
+  } = useApmPluginContext();
+  const licensePageUrl = useKibanaUrl(
```

Comment:
Would it be worth it to extract these two into a hook? Something like `useLicenseManagement` and you get `licensePageUrl` and `manageLicenseUrl` from there.

Reply:
`useApmPluginContext` and `useKibanaUrl` both are hooks, I don't think they will work as part of a hook as they should be used in a react component

Reply:
ah yes, good point!